content/en/Getting started/glossary.md

```diff
@@ -86,11 +86,13 @@ In the context of a Cobalt pentest, you can specify one of three options for an
 ## Finding
 
 A potential security flaw in an app or physical hardware. We include findings in vulnerability
-reports, as something that a threat actor can exploit. Examples:
+reports, as something that a threat actor can exploit.
 
-- [Cross-Side Scripting (XSS)](https://owasp.org/www-project-top-ten/2017/A7_2017-Cross-Site_Scripting_(XSS))
```

Comment:
From our discussion, I think this list is not accurate. With the `{{% finding-details %}} variable, I'm pulling the bulleted list from [here](https://github.com/cobalthq/cobalt-product-public-docs/pull/115/files#diff-9c302b93ee68e104d0b0567ea3ff8f9cb73dc7636019ef76fa7b4ba5ed1aabfbR1)